src/main/java/mate/project/exception/CustomGlobalExceptionHandler.java

```diff
@@ -48,6 +48,18 @@ public ResponseEntity<Object> handleEntityNotFoundException(
         return new ResponseEntity<>(body, HttpStatus.NOT_FOUND);
     }
 
+    @ExceptionHandler(RegistrationException.class)
+    public ResponseEntity<Object> handleRegistrationException(
+            RegistrationException ex,
+            WebRequest request) {
+        Map<String, Object> body = new LinkedHashMap<>();
+        body.put("timestamp", ZonedDateTime.now());
```

Comment:
The timestamp is set with the current timezone, which may lead to confusion if users are in different timezones. It would be better to use UTC to avoid this confusion.